### DISCUSSION TITLE
Application UI

### DISCUSSION THREAD

**Discussion by rehork** [ferraridamiano/ConverterNOW#187] [General]
Hi Damiano,

I've noticed that after recent update some important changes have been made. I mean small icons of units were hidden and show only when you enter any value for conversion. For me it's very important to see small icons when searching units or currencies. It works much faster to see the  icon then reading only a full name. I don't know if you have already received such comments. Nevertheless, I vote for returning back visibility of all units icons. Below I attach screenshots for comparison. 

![Screenshot_20220925-084825_Converter_NOW_084849](https://user-images.githubusercontent.com/67607997/192131949-be7ecd25-8630-49da-9089-fb4aa3cac4fd.png)
![Screenshot_20220925-085003538](https://user-images.githubusercontent.com/67607997/192131951-14729e16-cbb0-4f40-93a6-45883ace2a83.jpg)

**Comment by ferraridamiano** [comment]
Hi @rehork! I really appreciate this type of comments! I'm happy you reported this even if it means more work for me
What do you think if I make always visible the symbol of that unit? In this way you can see the same exact information of the previous UI with just a slighty different UI
I do not promise any ETA but I will take a look when as soon as I have time. In the meantime you can rollback to the previous version by [downloading it from github releases](https://github.com/ferraridamiano/ConverterNOW/releases/tag/v3.1.3)

**Reply by rehork** [reply]
You're welcome. I appreciate that you developing this application. Yes, please do so if it's easier for you this might work,too. Maybe you could bold the symbols, then they would be more visible. I'll stay with the newest release mainly because I translated your app into Polish language. Also it's good to have those symbols visible because some unit names were so "exotic" that I hardly found, hopefully proper localized names for them. So the symbol might clarify unit wanted.

**Reply by ferraridamiano** [reply]
Fixed!
You should be able to update the app in a few days (otherwise you can download it now from the release page)

**Reply by rehork** [reply]
Awesome! New release works as I would expect. Thank you!

**Comment by rehork** [comment]
Hello @ferraridamiano ,
I've just made some corrections to the translations. However, I found an issue probably caused by translated string which shouldn't be translated.
The source code is:
  "@reorderProperty":{
    "placeholders": {
      "property": {
        "type": "String",
        "example": "Length"
In PL I translated: "String" and "Length". To compare results (when choosing Length)
in EN - the tile is "Reorder Length" 
in PL - the title is "Reorder {property}"
It seems that it's not working as expected.

![Screenshot_20230213-223756_Converter_NOW](https://user-images.githubusercontent.com/67607997/218581464-62e0aa9b-4b5d-46e7-ac45-a0b5f2eab4fb.png)
![Screenshot_20230213-224307_Converter_NOW](https://user-images.githubusercontent.com/67607997/218581471-95b60d72-d290-4740-bde9-6cff3006cd79.png)

**Reply by rehork** [reply]
Above comment refers to ver. 3.2.1

**Reply by ferraridamiano** [reply]
Thank you for pointing this out, I will soon fix it!

**Reply by ferraridamiano** [reply]
Fixed! https://github.com/ferraridamiano/ConverterNOW/commit/a035a49a2a08c8c5feaf440527138be97a1aba87

**Reply by rehork** [reply]
Thanks!

**Comment by rehork** [comment]
On the other hand I've just upgraded to ver. 4.0.0 and on old Samsung ver. v7a (android 11) works fine, but on Mi A1 ver. v8a (android 11) currencies page doesn't load at all, rests is fine.

By the way, to be honest changing properties order in ver. 3.2.1 was less intuitive, but more elegant. In ver. 4.0.0 is the opposite, now it's more intuitive, but less elegant :-)))

**Reply by ferraridamiano** [reply]
See https://github.com/ferraridamiano/ConverterNOW/issues/211

Thank you for the feedback about the reordering layout, what do you like more? Do you have a better proposal?

**Reply by rehork** [reply]
You're welcome! I like the previous version, but it's just myself. Also in previous version when reordering for the first time it's not obvious what to do. It's great that when you touch a property there's a hint explaining what to do, that you have to touch and drag. Although, current users are rather smart enough to find out how to change order intuitively.
